_includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md

```diff
@@ -0,0 +1,102 @@
+## ActiveSpeakerObserver
+{: #ActiveSpeakerObserver}
+
+<section markdown="1">
+
+> `@inherits` [RtpObserver](#RtpObserver)
+
+An active speaker observer monitors the speech activity of the selected audio producers. It just handles audio producers (if [addProducer()](#rtpObserver-addProducer) is called with a video producer it will fail).
+
+Audio levels used for speech detection are read from an RTP header extension. No decoding of audio data is done. See [RFC6464](https://tools.ietf.org/html/rfc6464) for more information.
+
```

Comment:
Can we mention here that this class uses the algorithm of Jitsi with a link to the spec?

Reply:
For sure. I will mention it similar to how it is in the ActiveSpeakerObserver header file.

Reply:
Great.